dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -562,10 +562,15 @@ private Userinfoplus getUserInfo(String accessToken) {
     public Token addToken(@ApiParam("code") String satellizerJson) {
         Gson gson = new Gson();
         JsonElement element = gson.fromJson(satellizerJson, JsonElement.class);
-        JsonObject satellizerObject = element.getAsJsonObject();
-        final String code = getCodeFromSatellizerObject(satellizerObject);
-        final boolean registerUser = getRegisterFromSatellizerObject(satellizerObject);
-        return handleGitHubUser(null, code, registerUser);
+        if (element != null) {
+            JsonObject satellizerObject = element.getAsJsonObject();
```

Comment:
The code assumes the body is valid JSON and will fail with a 500 if it's not.

Trying your branch locally:

```
curl -X 'POST' \
  'http://localhost:4200/api/auth/tokens/github' \
  -H 'accept: application/json' \
  -H 'Authorization: Bearer <redacted>' -d 'hello'

{"code":500,"message":"There was an error processing your request. It has been logged (ID 10f155f24edf47cc)."
```
I know it wasn't the example in the ticket, but since you're here, could you please fix it too. Looks like you would need to catch a `JsonSyntaxException`. It might be hard to create a test for, unfortunately; I'm not sure the Swagger/OpenAPI client lets you create invalid requests like that.